### PR TITLE
W 10587060 add configure instuctions gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000lZdLeYAK/view

### CODE REVIEW

File: modules/ROOT/pages/_partials/task-configure-proxy.adoc [hunk 1]
```diff
@@ -127,6 +127,7 @@ You should now see the **TLS Context for Inbound Traffic** field enabled.
 . Click *Advanced Options*.
 . In *Proxy Version*, select *latest*. +
 This value ensures that your API proxy uses the latest released proxy version.
+If you update proxy version in exchange, return to this step to reconfigure your proxy to the latest version.
```
Comment (tutecano1995) [inline modules/ROOT/pages/_partials/task-configure-proxy.adoc]: This page is for proxy configuration, the proxy version is not the same as the API Asset Version in Exchange. I believe we can leave this page as it is.

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 2]
```diff
@@ -3,9 +3,19 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-After you create an API in Anypoint Platform, you configure the version details for the API and then publish it to Anypoint Exchange (Exchange). The API version resides in Exchange, and not in API Manager. However, you then manage this API version from API Manager.
+After an API is created, you can configure version details for the API and publish them to Anypoint Exchange (Exchange). The API asset version resides in Exchange, and not in API Manager. However, you manage the version of an API instance from API Manager. You can create multiple instances and expand the version information to view details about them.
```
Comment (tutecano1995) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]: I feel that the sentence is not completely clear.

```
After an API is created, you can configure version details for the API and publish them to Anypoint Exchange (Exchange)
```

Maybe we can remove the "After an API is created" part and just say something like:

```
You can configure the API Asset version details for the API and push it to Anypoint Exchange (Exchange).
```

Also maybe we can add some link to Exchange documentation. For example to this one:

https://docs.mulesoft.com/exchange/about-sharing-assets

Comment (tutecano1995) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]: Also we could expand this part in more details:
```
However, you manage the version of an API instance from API Manager
```
Something like:
```
However, once you [add a new API Instance in API Manager](https://docs.mulesoft.com/api-manager/2.x/create-instance-task), you can change the API Asset version of that API Instance.
```

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 3]
```diff
@@ -3,9 +3,19 @@ ifndef::env-site,env-github[]
...
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to configure the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
```
Comment (tutecano1995) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc:8]: I would not remove this part, maybe we can rephrase and clarify it a bit. But here is where we say that if they create a new API Instance linked to the same Exchange API Asset version, a completely new API Instance will be created in API Manager.

> You can create multiple instances and expand the version information to view details about them.

I'm not sure what does "expand the version information to view details about them" mean 🤔

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 4]
```diff
@@ -3,9 +3,19 @@ ifndef::env-site,env-github[]
...
+To manage the version of an API instance:
```
Comment (tutecano1995) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]: Maybe instead of saying "manage the version", I would say "change the asset version"

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 5]
```diff
@@ -3,9 +3,19 @@ ifndef::env-site,env-github[]
...
+
+. Go to *Anypoint Platform* > *API Manager*.
+. In *API Administration*, click the name of the API whose version you want to manage.
```
Comment (tutecano1995) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc:13]: Nice! This is much more clear than what we have now 🎉

**Review by tutecano1995 — COMMENTED** [review]
Nice work! Leave some suggestions, feel free to ask me any questions and change the wording

File: modules/ROOT/pages/_partials/task-configure-proxy.adoc (hunk 1 above)
Comment (glenn-rodgers-sf) [reply modules/ROOT/pages/_partials/task-configure-proxy.adoc]: I added this because the ticket is originally in reference to the configure page, but was really just providing information about normal API's. I don't think it hurts to have it here.

File: modules/ROOT/pages/manage-versions-instances-concept.adoc (hunk 2 above)
Comment (glenn-rodgers-sf) [reply modules/ROOT/pages/manage-versions-instances-concept.adoc]: I link to exchange in the bottom.

File: modules/ROOT/pages/manage-versions-instances-concept.adoc (hunk 3 above)
Comment (glenn-rodgers-sf) [reply modules/ROOT/pages/manage-versions-instances-concept.adoc:8]: This section still really confuses me as well. I feel like it's clear that creating a new API just creates a new instance then you manage this instance, but I don't know. Let me know what you think about this in the new one.

File: modules/ROOT/pages/_partials/task-configure-proxy.adoc (hunk 1 above)
Comment (tutecano1995) [reply modules/ROOT/pages/_partials/task-configure-proxy.adoc]: I believe that they were probably mixing concepts in the ticket. Users can't upload new proxy versions to exchange, we upload them internally and they just see new proxy versions available. Maybe we can rephrase this to something like:

```
If a new proxy version is available in Anypoint platform, you can return to this step to reconfigure your proxy to the latest version.
```

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 6]
```diff
@@ -3,9 +3,19 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-After you create an API in Anypoint Platform, you configure the version details for the API and then publish it to Anypoint Exchange (Exchange). The API version resides in Exchange, and not in API Manager. However, you then manage this API version from API Manager.
+You can configure version details for an existing API and publish them to Anypoint Exchange (Exchange). The API asset version resides in Exchange, and not in API Manager. However, you can create multiple instances of an API and manage their version from API Manager. See xref:create-instance-task.adoc[Adding an API Instance], for information on how to add an API instance to API Manager.
```
Comment (dhirsch1) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]:
```suggestion
You can configure version details for an existing API and publish them to Anypoint Exchange (Exchange). The API asset version resides in Exchange, and not in API Manager. However, you can create multiple instances of an API and manage their versions from API Manager. See xref:create-instance-task.adoc[Adding an API Instance], for information on how to add an API instance to API Manager.
```

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 7]
```diff
@@ -3,9 +3,19 @@ ifndef::env-site,env-github[]
...
+To change the asset version of an API instance:
+
+. Go to *Anypoint Platform* > *API Manager*.
+. In *API Administration*, click the name of the API whose version you want to manage.
++
+The API *Settings* detail appears:
++
+image::api-settings.png[API Manager administration settings.]
+. Select *Actions* > *Change API specification*.
+. In *Asset version*, Select latest.
```
Comment (dhirsch1) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]:
```suggestion
. In *Asset version*, select *latest*. 
```

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 8]
```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
...
+In Anypoint Exchange (Exchange), you can add, delete, and deprecate API versions, also known as _Exchange assets_. An API version in Exchange can have its own RAML file, API portal, and URL. See xref:exchange::manage-versions.adoc[Manage Versions] for more information.
 
-If you add new version details for an API definition and publish them to exchange, the version of your existing API instance does not change automatically. You must change the version of your API instance in API Manager if you want the instance to reflect the latest version. For more information on the differences between and API definition and an API instace, see xref:api-instance-landing-page.adoc[API Instance].
+If you add new version details for an API definition and publish them to exchange, the version of your existing API instance does not change automatically. You must change the version of your API instance in API Manager if you want the instance to reflect the latest version. For more information on API instaces, see xref:api-instance-landing-page.adoc[API Instance].
```
Comment (dhirsch1) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]: In line 8, clarify where you are adding the new version details:
" If you use API Manager (or do you use Exchange or both?) to add new version details..."

exchange > Exchange

Typo: instaces

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 9]
```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can add, delete, and deprecate API versions, also known as _Exchange assets_, in Anypoint Exchange (Exchange). An API version in Exchange can have its own RAML file, API portal, and URL. See xref:exchange::manage-versions.adoc[Manage Versions] for more information.
+In Anypoint Exchange (Exchange), you can add, delete, and deprecate API versions, also known as _Exchange assets_. An API version in Exchange can have its own RAML file, API portal, and URL. See xref:exchange::manage-versions.adoc[Manage Versions] for more information.
```
Comment (dhirsch1) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]: The way you construct the more information text should be consistent. Here you start it with the reference (See xref:exchange::manage-versions.adoc[Manage Versions] for more information.) Later on, you start it with "For more information on..."

It is better to start it with "For more information about..." because users can decide more quickly if they want to look at the reference. Kim wanted us to use "about" instead of "on."

**Review by dhirsch1 — DISMISSED** [review]

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 10]
```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Anypoint Exchange (Exchange), you can add, delete, and deprecate API versions, also known as _Exchange assets_. An API version in Exchange can have its own RAML file, API portal, and URL. See xref:exchange::manage-versions.adoc[Manage Versions] for more information on how to manage API versions in exchange.
+In Anypoint Exchange (Exchange), you can add, delete, and deprecate API versions, also known as _Exchange assets_. An API version in Exchange can have its own RAML file, API portal, and URL. For more information on how to manage API versions in exchange, see xref:exchange::manage-versions.adoc[Manage Versions].
```
Comment (dhirsch1) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]: For more information on > For more information about (to follow Kim's edits) (here and in the next paragraph)

exchange > Exchange (here and in the next paragraph)

**Review by tutecano1995 — DISMISSED** [review]

**Review by dhirsch1 — APPROVED** [review]
